app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/pairing/PairingViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package co.netguru.baby.monitor.client.feature.communication.pairing
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.ViewModel
+import io.reactivex.disposables.CompositeDisposable
+import java.net.URI
+import javax.inject.Inject
+import kotlin.random.Random
+import kotlin.random.nextInt
+
+class PairingViewModel @Inject constructor(
+    private val pairingUseCase: PairingUseCase
+) : ViewModel() {
+
+    internal val pairingCompletedState: LiveData<Boolean> = pairingUseCase.pairingCompletedState
+    private val compositeDisposable = CompositeDisposable()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/9259abfa8c2adcd3ca5c84779794ca9bdaf7e360/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/pairing/PairingViewModel.kt#L16"></span>Private property compositeDisposable is unused.</td>
    </tr>
  </tbody>
</table>